docs/languages/scala.md

```diff
@@ -35,7 +35,12 @@ M.config = function()
     excludedPackages = {},
   }
   metals_config.init_options.statusBarProvider = false
```

Comment:
seems like this capabilities is missing
```suggestion
  metals_config.capabilities = require("lvim.lsp").common_capabilities()
  metals_config.init_options.statusBarProvider = false
```

it might be easier to merge all of `require("lvim.lsp").get_common_opts()` instead

Reply:
Addressed.